client/src/views/Viewer/Viewer.vue

```diff
@@ -0,0 +1,488 @@
+                      :max-frame="maxFrame"
+                      :data="lineChartData"
+                    />
+                    <event-chart
+                      v-if="showTrackView && eventChartData"
+                      :start-frame="startFrame"
+                      :end-frame="endFrame"
+                      :max-frame="maxFrame"
+                      :data="eventChartData"
+                    />
+                  </template>
+                  <v-btn
+                    outlined
+                    x-small
+                    class="toggle-timeline-button"
+                    tab-index="-1"
+                    @click="showTrackView = !showTrackView"
+                  >
+                    {{ showTrackView ? "Detection" : "Track" }}
+                  </v-btn>
+                </Timeline>
+              </template>
+            </timeline-wrapper>
+          </template>
+          <annotation-layer
+            v-if="annotationData"
+            :data="annotationData"
+            :annotation-style="annotationStyle"
+            @annotation-click="annotationClick"
+            @annotation-right-click="annotationRightClick"
+          />
+          <edit-annotation-layer
+            v-if="editingTrackId !== null"
+            editing="rectangle"
+            :geojson="editingDetectionGeojson"
+            :feature-style="editingBoxLayerStyle"
+            @update:geojson="detectionChanged"
+          />
+          <edit-annotation-layer
+            v-if="featurePointing"
+            editing="point"
+            @update:geojson="featurePointed"
+          />
+          <text-layer
+            v-if="textData"
+            :data="textData"
+            :text-style="textStyle"
+          />
+          <marker-layer
+            v-if="markerData"
+            :data="markerData"
+            :marker-style="markerStyle"
+          />
+        </component>
+        <v-menu
+          v-if="selectedDetection"
+          offset-y
+        >
+          <template v-slot:activator="{ on }">
+            <v-btn
+              class="selection-menu-button"
+              icon
+              v-on="on"
+            >
+              <v-icon>mdi-dots-horizontal</v-icon>
+            </v-btn>
+          </template>
+          <v-list>
+            <v-list-item @click="toggleFeaturePointing('head')">
+              <v-list-item-title>
+                Add feauture points, starting with head (g key)
+              </v-list-item-title>
+            </v-list-item>
+            <v-list-item @click="toggleFeaturePointing('tail')">
+              <v-list-item-title>
+                Add feature points, starting with tail (t key)
+              </v-list-item-title>
+            </v-list-item>
+            <v-list-item @click="deleteDetection">
+              <v-list-item-title>
+                Delete both feauture points for current frame (q key)
+              </v-list-item-title>
+            </v-list-item>
+          </v-list>
```

Comment:
feauture -> feature

The last `@click`, should it be `deleteFeaturePoints` instead of `deleteDetection` ?